### PR TITLE
Start more profile tests

### PR DESCRIPTION
**Pull request by kehh**
Contains #19 and probably shouldn't.
let's double the number of tests ;)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "minimum-stability": "stable",
+    "license": "proprietary",
+    "authors": [
+        {
+            "name": "Kehan Harman",
+            "email": "kehan@gaiaresources.com.au"
+        }
+    ],
+    "require-dev": {
+        "phpunit/phpunit": "5.x"
```
Comment (leftclickben) [inline composer.json]: I was just treating phpunit as a global/external dependency.  As in, apt-get install phpunit.  Installing it per-project has always been troublesome.

File: tests/profile/ProfileUiTest.php [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+class ProfileUiTest extends PHPUnit_Framework_TestCase {
+    public function testProfileContainsUis() {
+        $basePath = dirname(dirname(__DIR__));
+        $doc = new DOMDocument();
+        $doc->load("$basePath/profile/rwahs.xml");
+        $xpath = new DOMXPath($doc);
+        $this->assertEquals(14, $xpath->query('//userInterfaces/userInterface')->length, 'The number of user interfaces should match');
```
Comment (leftclickben) [inline tests/profile/ProfileUiTest.php]: I reckon if we're going to bother with this test, we should check that all the expected UIs are present by checking the `code` for each one.  You could probably do this with XPath:

    $this->assertEquals(1, $xpath->query('//userInterfaces/userInterface[code="the_code"]'), 'The `the_code` user interface should exist');

**Comment by leftclickben** [comment]
This still appears to contain #19, seems to need a merge.

File: composer.json (hunk 1 above)
Comment (kehh) [inline composer.json]: It was mainly so PHPStorm would automatically phpunit as an external library so I could be lazy and get it to autocomplete. I know I could have manually added the library, but it seemed that this is something that a dependency manager should be used for.

File: tests/profile/ProfileUiTest.php (hunk 2 above)
Comment (kehh) [inline tests/profile/ProfileUiTest.php]: Ok, I've done that, and remove an unused UI.

**Pull request merged**